ggml/CMakeLists.txt

```
@@ -74,10 +74,10 @@ if (NOT GGML_CUDA_GRAPHS_DEFAULT)
 endif()
 
 # general
-option(GGML_STATIC "ggml: static link libraries"         OFF)
-option(GGML_NATIVE "ggml: enable -march=native flag"     ${GGML_NATIVE_DEFAULT})
-option(GGML_LTO    "ggml: enable link time optimization" OFF)
-option(GGML_CCACHE "ggml: use ccache if available"       ON)
+option(GGML_STATIC "ggml: static link libraries"                  OFF)
+option(GGML_NATIVE "ggml: optimize the build for the current CPU" ${GGML_NATIVE_DEFAULT})
```

`GGML_NATIVE` may also be used by other backends, e.g. ggml-cuda uses it to build for the native GPU.
```suggestion
option(GGML_NATIVE "ggml: optimize the build for the current system" ${GGML_NATIVE_DEFAULT})
```